pkg/plugins/generic/generic_plugin.go

```diff
@@ -81,7 +81,8 @@ type genericPluginOptions struct {
 	skipBridgeConfiguration bool
 }
 
-const scriptsPath = "bindata/scripts/kargs.sh"
+const daemonScriptsPath = "bindata/scripts/kargs.sh"
```

Comment:
nit. try to refrain of relative path
```suggestion
const daemonScriptsPath = "/bindata/scripts/kargs.sh"
```

Reply:
done